launch/sensing_launch/launch/sensing.launch.xml

```diff
@@ -0,0 +1,26 @@
+<launch>
+
+  <arg name="launch_driver" default="true" description="do launch driver"/>
+  <arg name="sensor_model" default="aip_xx1" description="sensor model name"/>
+  <arg name="vehicle_param_file" default="$(find-pkg-share vehicle_info_util)/config/vehicle_info.param.yaml" description="path to the file of vehicle info yaml"/>
```

Comment:
@1222-takeshi 
It's necessary to add vehicle_info_util package as exec depend

Reply:
fixed https://github.com/autowarefoundation/autoware.universe/pull/165/commits/9fe36debcfd418f211425bc7b184617b0027a00f